gtsfm/utils/test_ellipsoid_utils.py

```diff
@@ -0,0 +1,121 @@
+"""Unit tests for functions in ellipsoid utils file.
+
+Authors: Adi Singh
+"""
+import gtsfm.utils.ellipsoid as ellipsoid_utils
+import numpy as np
+import numpy.testing as npt
+import unittest
+
+
+class TestEllipsoidUtils(unittest.TestCase):
+    def test_center_point_cloud_sample_points(self):
+        """Testing the center_point_cloud() function with 5 sample points."""
+        sample_points = [[6, 13, 9], [5, 9, 7], [6, 10, 9], [9, 13, 6], [5, 12, 9]]
+        computed = ellipsoid_utils.center_point_cloud(sample_points)
+        expected = np.array([[-0.2, 1.6, 1], [-1.2, -2.4, -1], [-0.2, -1.4, 1], [2.8, 1.6, -2], [-1.2, 0.6, 1]])
+        npt.assert_almost_equal(computed, expected, 6)
+
+    def test_center_point_cloud_wrong_dims(self):
+        """Testing the center_point_cloud() function with 5 sample points of 2 dimensions."""
+        sample_points = [[6, 13], [5, 9], [6, 10], [9, 13], [5, 12]]
```

Comment:
You could also add a test for some input like [1,2,3] after you handle invalid dimensions ;)